config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -272,8 +266,6 @@ def presubmit_test(branch='master',
         env['CLOUD_PROVIDER'] = cloud
         env['CLUSTER_NAME'] = f"e2e-{name_hash[0:10]}-{name_hash[11:16]}.test-cncf-aws.k8s.io"
         env['KOPS_STATE_STORE'] = 's3://k8s-kops-prow'
-        if irsa and cloud == "aws":
-            env['KOPS_IRSA'] = "true"
```

Comment:
Scenarios ignore `extra_flags`, so `KOPS_IRSA` is needed.

Reply:
Thanks, somehow I missed it. Reverting.